src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Experimenting with QuicTransport
+subhead: QuicTransport is a new API offering low-latency, bi-directional, client-server messaging. Learn more about its use cases, and how to give feedback about the future of the implementation.
+authors:
+  - jeffposnick
+description: QuicTransport is a new API offering low-latency, bi-directional, client-server messaging. Learn more about its use cases, and how to give feedback about the future of the implementation.
+date: 2020-06-03
+updated: 2020-06-05
+hero: hero.jpg
+hero_position: center
+alt: |
+  Photo of fast-moving traffic.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/-6744140441987317759
+tags:
+  - blog
+  - capabilities
+  - quic
+  - networking
+  - udp
+  - webtransport
+  - behind-a-flag
```

Comment:
wanted to mention that I think some of these tags aren't in postTags.js and thus won't show up on the site.

Reply:
Ah, okay. I'll prune.